modules/ding_provider/connie_search/connie_search.test

```diff
@@ -128,4 +128,18 @@ class ConnieSearchSearchProviderImplementationTestCase extends DingSearchProvide
   public function getDummyRelation() {
     return new TingRelation('dummytype', 'http://example.com/dummy', (object) []);
   }
+
+  /**
+   * Returns the list of all common fields.
+   *
+   * @return string[]
+   *   List of fields.
+   */
+  public function getExpectedCommonFields() {
+    $class = new ReflectionClass('\Ting\Search\TingSearchCommonFields');
+    // Return all static properties that does not start with _.
```

Comment:
Another approach would be to let the "Enum" provide a function which will return all legal values. I actually prefer that.

Reply:
I have a single _ field in there for holding the prefix for the fields - but I like the idea of just having the class itself enumerate the list of possible values way better, it will make a number of other cases much cleaner.